ViewPumpSample/src/main/java/io/github/inflationx/viewpump/sample/TextField.java

```diff
@@ -0,0 +1,16 @@
+package io.github.inflationx.viewpump.sample;
+
+import android.content.Context;
+import android.util.AttributeSet;
+import android.widget.TextView;
+
+/**
+ * For ViewPump.
+ */
+public class TextField extends TextView {
```

Comment:
Again I think unless you want to use this, should stay part of Calligraphy sample?